lib/easypost/client.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require_relative 'services'
+require_relative 'client_configuration'
+require_relative 'http_client'
+require 'json'
+
+class EasyPost::Client
+  def initialize(api_key, api_base = nil, api_version = nil, read_timeout = 60, open_timeout = 30)
+    @configuration = EasyPost::ClientConfiguration.new(
+      api_key, api_base, api_version, read_timeout,
+      open_timeout,
+    )
```

Comment:
nit: prefer each of these either on ones line or since there are many of them, prefer them each on separate lines.